index.html

```diff
@@ -21,13 +19,14 @@
         </div>
     </header>
     <section class="headline">
-
         <div class="container1">
             <h1 class="head">I'm Marianne <br/>
-            Glad to see you!</h1>
-       
+            Glad to see you!</h1> 
         <p class="desc">
-            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            I’m a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t 
+            hestiate to contact me.
```

Comment:
- [ ] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).